app/services/trending_groups.rb

```diff
@@ -0,0 +1,26 @@
+class TrendingGroups
```

Comment:
We shouldn't be duplicating code between TrendingAnime and TrendingGroups like this. Extract the common functionality into a common parent class or something.